packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -100,6 +100,15 @@ export async function findSDKConnectionsByOrganization(organization: string) {
   return docs.map(toInterface);
 }
 
+export async function findSDKConnectionsByOrganizations(
+  organizations: string[]
+) {
+  const docs = await SDKConnectionModel.find({
+    organizations,
+  });
+  return docs.map(toInterface);
+}
```

Comment:
This Mongo query is broken (no field called `organizations`), but I don't think we need this function at all if we make the change to listSdkConnections

Reply:
oops, updated